ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -155,12 +155,27 @@ then
 		then
 			set USE_IO no
 		else
-			set USE_IO yes
+			if ver hwcmp MINDPX_V2
+			then
+				set USE_IO no
+			else
+				set USE_IO yes
+			fi
 		fi
 	else
 		set USE_IO no
 	fi
 
+	# should set to 0.8 for mindpx-v2 borad.
```

Comment:
We are deprecating INAV right now, so you will want to invest some effort into testing LPE or EKF2.